docs/install.md

```diff
@@ -1,26 +1,137 @@
+
+Helper command for setting up k0s as controller node on a brand-new system. Must be run as root (or with sudo)
+
+Usage:
+  k0s install controller [flags]
+
+Aliases:
+  controller, server
+
+Examples:
+All default values of controller command will be passed to the service stub unless overriden. 
+
+With controller subcommand you can setup a single node cluster by running:
+
+        k0s install controller --enable-worker
+
+
+Flags:
+  -c, --config string            config file (default: ./k0s.yaml)
+      --cri-socket string        contrainer runtime socket to use, default to internal containerd. Format: [remote|docker]:[path-to-socket]
+  -d, --debug                    Debug logging (default: false)
+      --enable-worker            enable worker (default false)
+  -h, --help                     help for controller
+  -l, --logging stringToString   Logging Levels for the different components (default [konnectivity-server=1,kube-apiserver=1,kube-controller-manager=1,kube-scheduler=1,kubelet=1,kube-proxy=1,etcd=info,containerd=info])
+      --profile string           worker profile to use on the node (default "default")
+      --token-file string        Path to the file containing join-token.
+
+Global Flags:
+      --data-dir string        Data Directory for k0s (default: /var/lib/k0s). DO NOT CHANGE for an existing setup, things will break!
+      --debugListenOn string   Http listenOn for debug pprof handler (default ":6060")
+```
+
+For example, the command below will install a single node k0s service on Ubuntu 20.10:
+
+```
+$ k0s install controller --enable-worker
+INFO[2021-02-24 11:05:42] no config file given, using defaults         
+INFO[2021-02-24 11:05:42] creating user: etcd                          
+INFO[2021-02-24 11:05:42] creating user: kube-apiserver                
+INFO[2021-02-24 11:05:42] creating user: konnectivity-server           
+INFO[2021-02-24 11:05:42] creating user: kube-scheduler                
+INFO[2021-02-24 11:05:42] Installing k0s service
+```
+
+## Run k0s as a service
+
+```sh
+$ systemctl start k0scontroller
+```
+
+### Check service status
+
+```sh
+# systemctl status k0scontroller.service
+     Loaded: loaded (/etc/systemd/system/k0scontroller.service; enabled; vendor preset: enabled)
+     Active: active (running) since Fri 2021-02-26 08:37:23 UTC; 1min 25s ago
+       Docs: https://docs.k0sproject.io
+   Main PID: 1408647 (k0s)
+      Tasks: 96
+     Memory: 1.2G
+     CGroup: /system.slice/k0scontroller.service
+     ....
+
+### Query cluster status
+
+```sh
+$ k0s status
+Version: v0.11.0-beta.2-16-g02cddab
+Process ID: 9322
+Parent Process ID: 1
+Role: controller+worker
+Init System: linux-systemd
+```
+
+### Fetch nodes
+```sh
+$ k0s kubectl get nodes
+NAME   STATUS   ROLES    AGE    VERSION
+k0s    Ready    <none>   4m6s   v1.20.4-k0s1
+```
+
+## Under the hood
+
+The command will create service stub file which can be found at:
+
+```
+$ cat /etc/systemd/system/k0scontroller.service 
+```
+
```

Comment:
```suggestion
```

Just get rid of the section. The file location is different for openrc anyway.

Reply:
Agreed